<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PublishDlg</class>
 <widget class="QDialog" name="PublishDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Publish</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Use versions of xrefs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbUseVersion">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Use plot style</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbPlotStyles">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Publish to</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QCheckBox" name="cbPDF">
        <property name="text">
         <string>PDF</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbDWF">
        <property name="text">
         <string>DWF</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="cbPLT">
          <property name="text">
           <string>PLT using device</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbDevices">
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbNotScale">
     <property name="text">
      <string>Don't change scale</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Output directory</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="leDirName"/>
       </item>
       <item>
        <widget class="QToolButton" name="tbTreeSel_2">
         <property name="icon">
          <iconset resource="resource1.qrc">
           <normaloff>:/some/ico/ico/dropdownbutton.png</normaloff>:/some/ico/ico/dropdownbutton.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource1.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PublishDlg</receiver>
   <slot>Accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PublishDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbPDF</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PublishDlg</receiver>
   <slot>CheckAnySelected(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbDWF</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PublishDlg</receiver>
   <slot>CheckAnySelected(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbPLT</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PublishDlg</receiver>
   <slot>CheckAnySelected(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Accept()</slot>
  <slot>CheckAnySelected(int)</slot>
 </slots>
</ui>
